sql/telemetry_derived/italy_covid19_outage_v1/query.sql

```diff
@@ -0,0 +1,292 @@
+DECLARE country_code STRING DEFAULT "IT";
+
+DECLARE fromdate DATE DEFAULT '2020-01-01';
+
+DECLARE todate DATE DEFAULT '2020-03-31';
+
+-- The minimum size of a bucket for it to be published.
+DECLARE MIN_NUMBER_CLIENTS INT64 DEFAULT 5000;
```

Comment:
Having these DECLARE statements turns this from a simple query into a BQ script. Our tooling for publishing public data is not set up to handle that, since script jobs cannot be run with a destination table set.

I think it would be best to turn this into a simple query by removing these DECLAREs and instead inserting the literal values into the query, even though that means duplicating the literal values.

As BQ scripting matures, I hope we will be able to support this syntax in bigquery-etl in the future.